src/pybind/mgr/orchestrator.py

```diff
@@ -384,36 +386,158 @@ class DriveGroupSpec(object):
     """
     Describe a drive group in the same form that ceph-volume
     understands.
+    This structure allows the selection of storage devices
+    in a set of nodes
     """
-    def __init__(self, devices):
-        self.devices = devices
+    def _init_attribute(self, att_dict, attribute_name, default_value):
+        """ Extract attribute from dict, if not present applies default value
+
+        :param att_dict      : dictionary with attributes
+        :param attribute_name: The attribute to initialize
+        :param default_value : Value applied if the attribute is not in the dict
+        """
+        if attribute_name in att_dict.keys():
+            value = att_dict[attribute_name]
+        else:
+            value = default_value
```

Comment:
you can replace this with `att_dict.get(attribute_name, default_value)`

Reply:
Sure!! Thx!!